src/contracts/exchanges/thirdparty/0x/Exchange.sol

```diff
@@ -19,9 +19,9 @@
 pragma solidity ^0.4.21;
 pragma experimental ABIEncoderV2;
 
-import "../../../dependencies/token/StandardToken.sol";
-import "./LibOrder.sol";
-import "./Ownable.sol";
+import "StandardToken.sol";
+import "LibOrder.sol";
+import "Ownable.sol";
 
```

Comment:
maybe we should just rename this file to ZeroExExchange, since importing `Exchange.sol` is not totally clear

Reply:
Yeah, I thought about that too. But ABI files get named after the actual contract name, not the file name holding the contract. But I think we should not change the actual contract names of 3rd party contracts.

Reply:
Yeah fair enough